### DISCUSSION TITLE
LM Studio's AI base recommendation

### DISCUSSION THREAD

**Discussion by MadayLyncoln** [SubtitleEdit/subtitleedit#8526] [General]
Good evening, which LM Studio AI base do you recommend for translating subtitles, especially bluray subtitles in .ass which are full of codes?

sorry for my English

**Comment by TobbyTravel** [comment]
Llama-3 70B, it works great with Ollama also, it's easier to setup and no configuration needed as LM_studio can be overwhelming.

**Reply by MadayLyncoln** [reply]
Thanks, I'll test it here.

**Comment by MadayLyncoln** [comment]
Good afternoon, I checked Subtitle Edit and when using Ollama it doesn't have the option to use Llama-3 70B, only Llama-3. I saw that it has Llama2:70B.

**Comment by TobbyTravel** [comment]
You can manually write the model name you want to use in the newest version
of subtitle editor.
